pkg/util/k8s/k8s.go

```diff
@@ -0,0 +1,23 @@
+package k8s
+
+import (
+	"context"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/rest"
+	"k8s.io/client-go/tools/clientcmd"
+)
+
+func GetClientSet(ctx context.Context) (*kubernetes.Clientset, error) {
```

Comment:
Let's use the controller-runtime client, it reads from a cache and you supports all runtime.Objects. `LVMClusterReconciler.Client.Create/Update/Delete`